.github/workflows/tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           CC: "sccache clang"
           CXX: "sccache clang++"
       - name: Run tests
-        run: cargo test --workspace --all-features --all-targets
+        run: RUST_MIN_STACK=8388608 cargo test --workspace --all-features --all-targets
```

Comment:
what happened here? any idea why?

Reply:
`test_loading_static_value` requires this